meta-phosphor/common/recipes-phosphor/host-ipmid/host-ipmid.bb

```diff
@@ -31,3 +34,16 @@ do_install() {
         install -m 0755 -d ${D}${includedir}/host-ipmid
         install -m 0644 ${S}/ipmid-api.h ${D}${includedir}/host-ipmid/
 }
+
+def find_cfgs(d):
+    sources=src_patches(d, True)
+    sources_list=[]
+    for s in sources:
+        if s.endswith('.conf'):
+            sources_list.append(s)
+
+    return sources_list
+
+do_configure () {
+        ${S}/generate_whitelist.sh ${@" ".join(find_cfgs(d))}  > ipmiwhitelist.C
```

Comment:
You should be able to just set an environment variable to all of the .conf files instead of running the generate_whitelist script directly.